exec/totemconfig.c

```diff
@@ -74,6 +74,7 @@
 #define FAIL_TO_RECV_CONST			2500
 #define	SEQNO_UNCHANGED_CONST			30
 #define MINIMUM_TIMEOUT				(int)(1000/HZ)*3
+#define MINIMUM_TIMEOUT_HOLD				7*MINIMUM_TIMEOUT/15
```

Comment:
How is this one computed? Could it be written as an computation instead of hardcoded value? What is 7 and 15?